### PR TITLE
[13.0][MIG] crm

### PR DESCRIPTION
**Pull request by Yasaie**
Migration of `crm`
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+crm          / crm.team                 / dashboard_graph_group_pipeline (selection): DEL selection_keys: ['day', 'month', 'stage', 'user', 'week']
+crm          / crm.team                 / dashboard_graph_model (False) : DEL selection_keys: ['crm.lead'], mode: modify
+# NOTHING TO DO
+crm          / res.partner              / team_id (False)               : DEL mode: modify
+crm          / res.users                / team_id (False)               : DEL mode: modify
+# NOTHING TO DO: wrong analysis
+
+---XML records in module 'crm'---
+DEL crm.team: sales_team.team_sales_department (noupdate)
+# DONE: post-migration removed
```
Comment (MiquelRForgeFlow) [inline addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt]: `# NOTHING TO DO: Don't remove it, still in sales_team module`

File: addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+crm          / crm.lead                 / phone_state (selection)       : NEW selection_keys: ['correct', 'incorrect'], isfunction: function, stored
+# NOTHING TO DO
+crm          / crm.lead.scoring.frequency / lost_count (float)            : NEW
+crm          / crm.lead.scoring.frequency / team_id (many2one)            : NEW relation: crm.team
+crm          / crm.lead.scoring.frequency / value (char)                  : NEW
+crm          / crm.lead.scoring.frequency / variable (char)               : NEW
+crm          / crm.lead.scoring.frequency / won_count (float)             : NEW
+crm          / crm.lead.scoring.frequency.field / field_id (many2one)           : NEW relation: ir.model.fields, required
+# NOTHING TO DO: New models
+crm          / crm.stage                / is_won (boolean)              : NEW
```
Comment (MiquelRForgeFlow) [inline addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt:24]: Put is_won next to probability. In pre-migration: copy-column of probability using rename to None. In post-migration: set "is_won" = True if "stage.probability" = 100

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (Yasaie) [reply addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt]: changed

File: addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 2 above)
Comment (Yasaie) [reply addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt:24]: Thanks. changed

File: addons/crm/migrations/13.0.1.0/post-migration.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        WHERE {column} = 100
+        """.format(
+            column=openupgrade.get_legacy_name('probability')
+        ),
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    fill_crm_stage_is_won(cr)
```
Comment (kos94ok-3D) [inline addons/crm/migrations/13.0.1.0/post-migration.py]: env.cr

**Review by kos94ok-3D — CHANGES_REQUESTED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
please rebase

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by Abranes — APPROVED** [review]

**Comment by Abranes** [comment]
Tested, it works!

**Pull request merged**